CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ PrestaShop 1.6.x and 1.7.x
 
 # Changelog
 
+## [3.3.5] - 2021-06-14
+### Fixed
+- Hide CVV field when it is optional
```

Comment:
The CVV is either enabled or disabled, never optional.

Reply:
Changelog updated